governance/handler.go

```diff
@@ -442,6 +442,5 @@ func (gov *Governance) GetGovernanceItemAtNumber(num uint64, key string) (interf
 }
 
 func (gov *Governance) GetLatestGovernanceItem(key int) interface{} {
-	ret, _ := gov.currentSet.GetValue(key)
-	return ret
+	return gov.GetGovernanceValue(key)
```

Comment:
If it is a simple alias, how about deleting the `GetLatestGovernanceItem()`?

Reply:
@jeonghonah I also considered that. Though I wanted to make little change as much as possible until the remaining patches are applied, the function is called by just three other functions. So I changed those callers to call `GetGovernanceValue` instead of `GetLatestGovernanceItem`. 
Please take another look. =)